CONTRIBUTING.md

```diff
@@ -0,0 +1,92 @@
+Creating a New Example Implementation
+============
+
+Getting Started
+---
+1. [Fork this repo](https://github.com/ericelliott/todotasks/fork)
+
+2. clone your fork and copy the `base` files
+    ```
+    # clone!
+    git clone https://github.com/<your-fork>/todotasks.git
+    # make a copy of the `base` implementation:
+    cp -r todotaks/implementations/base todotasks/implementations/<your-build-process-name>
+    # install dependencies
+    cd todotasks/implementations/<your-build-process-name>
+    npm install
+    ```
+
+3. Implement all the tasks listed below in your selected build process and link them to npm scripts of the same name  in `package.json`.
+
+4. Update the `README.md` with detailed information about your task runner. See below for details.
+
+5. Verify that your example covers all the below tasks by running this helper script:
+    ```
+    // TODO: create a script that will do some simple validation/sanity checking of examples... basically excercise all the npm scripts.
+    ```
+6. Commit your changes (preferably in a single revision) and push them to your fork.
+
+7. Send a pull request for review.
+
+Tasks
+---
+Your implementation example must implement all of the tasks below. Each task should be mapped to an `npm` script of the same name.
+
+Example `package.json`:
+```
+{
+  "name": "todotasks-template-gulp",
+  "version": "1.0.0",
+  "description": "",
+  "scripts": {
+    "clean": "gulp scrubadub"
+    "lint": "gulp lintme",
+    "build": "gulp doit",
+    "dev": "gulp watchmystuff"
+  },
+  "author": "",
+  "license": "ISC"
+}
+```
+
+**clean**
+Delete all files in the `dist` directory.
+
+**lint**
+Run `eslint` on all js files in the `js` directory.
+
+**css**
+Use node-sass to compile `bower_components/todomvc-common/*.css`and output to `dist/app.css`.
+
+**uglify**
+Take all JavaScript  from `/src/plainjs/*`and generate a single concatenated+uglify'd file at `dist/ugify.js`. Please use *uglfy* for this task.
```

Comment:
I'm using `minifyify` to preserve sourcemaps, but I can change that. Also, why not call it `dist/app.js`.